timeMachine.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
+var timeMachine = ( function iife() {
+
+  var month = new Array( 12 ) ;
```

Comment:
While this technically works fine, using array literals is always preferred over instantiating new array instances. This means something like this:

`var month = ['January', 'February', 'March', 'April', ...];`